core/src/test/scala/org/broadinstitute/dsde/rawls/monitor/GoogleGroupSyncMonitorSpec.scala

```diff
@@ -53,13 +53,15 @@ class GoogleGroupSyncMonitorSpec(_system: ActorSystem) extends TestKit(_system)
     val workerCount = 10
     system.actorOf(GoogleGroupSyncMonitorSupervisor.props(10 milliseconds, 0 milliseconds, pubsubDao, topic, "subscription", workerCount, userServiceConstructor))
 
-    val testGroups = (for(i <- 0 until workerCount*4) yield RawlsGroupRef(RawlsGroupName(s"testgroup_$i")))
-    pubsubDao.publishMessages(topic, testGroups.map(_.toJson.compactPrint))
+    // GoogleGroupSyncMonitorSupervisor creates the topic, need to wait for it to exist before publishing messages
+    awaitCond(pubsubDao.topics.contains(topic), 10 seconds)
```

Comment:
default behaviour is to poke to see if the condition is true every 100ms -- you ok with this?